dags/burnham.py

```diff
@@ -0,0 +1,147 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import datetime
+
+from airflow import models
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from operators.bq_sensor import BigQuerySQLSensorOperator
+from operators.gcp_container_operator import GKEPodOperator
+
+DEFAULT_GCP_CONN_ID = "google_cloud_derived_datasets"
+DEFAULT_GKE_LOCATION = "us-central1-a"
+DEFAULT_GKE_CLUSTER_NAME = "bq-load-gke-1"
+DEFAULT_GKE_NAMESPACE = "default"
+
+DAG_OWNER = "rpierzina@mozilla.com"
+DAG_EMAIL = ["telemetry-alerts@mozilla.com", "rpierzina@mozilla.com"]
+
+BURNHAM_PLATFORM_URL = "https://incoming.telemetry.mozilla.org"
+
+
+def new_burnham_operator(
```

Comment:
This should probably be factored out to a `dags/util/burnham.py` module to match style elsewhere. We generally keep any significant code outside of the DAG itself.

Reply:
Done! ✅